graph/resolvers/root_mutation.js

```diff
@@ -1,7 +1,13 @@
 const RootMutation = {
-  createComment: async (_, {input}, {mutators: {Comment}}) => ({
-    comment: await Comment.create(input),
-  }),
+  createComment: async (_, {input}, {mutators: {Comment}, loaders: {Actions}}) => {
+    const comment = await Comment.create(input);
+
+    // Retrieve flags that was assigned to comment.
+    const actions = await Actions.getByID.load(comment.id);
+    const flags = actions.filter(({action_type}) => action_type === 'FLAG');
+
+    return {comment, flags};
```

Comment:
Too specific, this should instead be `actions`, don't filter out the flags.

Reply:
You said something with `filter` in the call, so I was thinking what possibly could I filter and ended up filtering out the flags 😅

Reply:
can revert it 👍